### PR TITLE
Added section about disabling already loaded actions in a controller

### PR DESCRIPTION
**Pull request by Spriz**
I havn't compiled this, so I don't know if it looks correct - otherwise I'm sure somebody could point me where to go :smile:

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -114,6 +114,32 @@ An example of on the fly enabling an Crud action with configuration
 
   }
 
+Disabling loaded actions
+----------------
```
Comment (jippi) [inline docs/configuration.rst]: the `-` need to cover the entire length of the title

File: docs/configuration.rst [hunk 2]
```diff
@@ -114,6 +114,32 @@ An example of on the fly enabling an Crud action with configuration
...
+    public $components = [
+      'Crud.Crud' => [
+        'actions' => ['Crud.Index', 'Crud.View', 'Crud.Delete', 'Crud.Edit']
+      ]
+    ];
+
+  }
+
+..code-block:: phpinline
+  class AppController extends \Cake\Controller\Controller {
```
Comment (jippi) [inline docs/configuration.rst]: this one should not be AppController - but an instance extending the AppController

File: docs/configuration.rst [hunk 3]
```diff
@@ -114,6 +114,32 @@ An example of on the fly enabling an Crud action with configuration
...
+
+    public function beforeFilter(\Cake\Event\Event $event) {
+      $this->Crud->disable(['Edit', 'Delete']);
```
Comment (jippi) [inline docs/configuration.rst]: should also call parent :)

Comment (Spriz) [inline docs/configuration.rst]: Fixed

File: docs/configuration.rst (hunk 2 above)
Comment (Spriz) [inline docs/configuration.rst]: Fixed.

File: docs/configuration.rst (hunk 1 above)
Comment (Spriz) [inline docs/configuration.rst]: Fixed.

**Pull request merged**